src/pages/FindWorkflow/FindWorkflowModal.js

```diff
@@ -67,19 +70,25 @@ const suggestedWorkflowsList = [
 const FindWorkflowModal = ({ onDismiss }) => {
   const [selectedSubHeader, setSelectedSubHeader] = useState('browse-suggested-workflows')
   const [loading, setLoading] = useState()
+  const [workflowName, setWorkflowName] = useState('')
+  const [versionName, setVersionName] = useState('')
+  const [methodUrl, setMethodUrl] = useState('')
 
   const signal = useCancellation()
 
-  const submitMethod = withBusyState(setLoading, async ({ method }) => {
+  const submitMethod = withBusyState(setLoading, async method => {
     try {
+      const rawGithubUrl = reconstructToRawUrl(method.method_url)
```

Comment:
To clarify my understanding: what happens if the `method_url` is already a raw github url? 🤔

Reply:
It still works. My understanding of the find and replace is that if it finds the substring, it'll replace it, if not, nothing will happen and the url remains unchanged.